packages/cinterop/src/jvm/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt

```diff
@@ -180,22 +180,26 @@ actual object RealmInterop {
         realmc.realm_config_set_in_memory(config.cptr(), inMemory)
     }
 
-    actual fun realm_open(config: RealmConfigurationPointer, dispatcher: CoroutineDispatcher?): Pair<LiveRealmPointer, Boolean> {
+    actual fun realm_create_scheduler(): RealmSchedulerPointer =
+        LongPointerWrapper(realmc.realm_scheduler_make_default(), false)
```

Comment:
Making it managed, we will release it manually when closing realms.

Reply:
But why is it manually managed? Seems way better to make it managed by the GC as we can then either `close`/`release` it explicitly or the GC will do it if we forget/can't. Currently this will not be release in the `MemoryTests.garbageCollectorShouldFreeNativeResources` where we don't do an explicit `Realm.close`. (Or maybe it will now with this PR? ... This was where the whole PR started 🙈)